pkg/utils/parse_arf_result.go

```diff
@@ -506,6 +622,37 @@ func handleDependencyAnnotation(u *unstructured.Unstructured, annotations map[st
 	return annotations
 }
 
+func handleValueUsed(valuesList []string, annotations map[string]string) map[string]string {
+
+	annotations[compv1alpha1.RemediationValueUsedAnnotation] = strings.Join(valuesList, ",")
+
+	return annotations
+}
+
+func handleNotFoundValue(notFoundValues []string, annotations map[string]string) map[string]string {
+
+	annotations[compv1alpha1.RemediationUnsetValueAnnotation] = strings.Join(notFoundValues, ",")
```

Comment:
Oh, my earlier suggestion for changing RemediationUnsetVariableAnnotation might not be right, but instead change the annotation name to `unset-value`?